### PR TITLE
add possibility to export svg to png/jpeg

### PR DESCRIPTION
**Pull request by apryka**
## Related tickets?
https://github.com/keen/keen-dataviz.js/issues/134
## How should this be tested?

Step through the code line by line. Things to keep in mind as you review:
 - Are there any edge cases not covered by this code?
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:

- [ ] Does the code compile without warnings (check shell, console)?
- [ ] Do all tests pass?
- [ ] Does the UI, pixel by pixel, look exactly as expected (check various screen sizes, including mobile)?
- [ ] If the feature makes requests from the browser, inspect them in the Web Inspector. Do they look as expected (parameters, headers, etc)?
- [ ] If the feature sends data to Keen, is the data visible in the project if you run an extraction (include link to collection/query)?
- [ ] If the feature saves data to a database, can you confirm the data is indeed created in the database?

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -826,6 +827,10 @@ Dataviz.prototype.width = function(num){
   return this;
 };
 
+Dataviz.prototype.exportImage = function(node, quality, bgcolor) {
+  exportImage(node, quality, bgcolor);
+};
+
```
Comment (adamkasprowicz) [inline lib/index.js]: let's use config objects if possible, like:
```javascript
Dataviz.prototype.exportImage = function({ quality = 1, bgcolor = '#fff' } = {}) {
  exportImage({
   node: this.config.containerElement,
   quality,
   bgcolor
  });
};
```
